homework-submission/todo-app/src/component/dynamicComponent/dynamicDescription.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+const DynamicDescription = ({ description }) => <div>description: {description}</div>;
```

Comment:
I'm not sure these all needed to be their own components, but great job breaking it down!

Reply:
I tried to make them totally ‘static’.